README.md

```diff
@@ -110,6 +110,50 @@ curl localhost:8080
 # Output: Hello World from a PHP HTTP function!
 ```
 
+## Run your function on Google Cloud Functions
```

Comment:
Nit: worth linking to the [official GCP docs](https://cloud.google.com/functions/docs/deploying)?

(That page doesn't mention PHP...at least not yet.)

Reply:
Added